CONTRIBUTING.md

```diff
@@ -2,6 +2,8 @@
 
 Thank you for your interest in contributing to SideStore! SideStore is a community driven project, and it's made possible by people like you.
 
+By contributing to this Project (SideStore), you agree to the Developer's Certificate of Origin found in [AGREEMENT.md](AGREEMENT.md). Any contributions to this project after the addition of the Developer's Certificate of Origin are subject to its policy.
```

Comment:
You could've made the file name more descriptive. I don't know how other projects do this but I'd rather the file name to be descriptive

Reply:
Something more along the lines of  `CERTIFICATE-OF-ORIGIN.md`